src/Components/App/App.test.js

```diff
@@ -7,564 +7,697 @@ import { VideoProvider } from '../../Context/VideoContext'
 import { SettingsProvider } from '../../Context/SettingsContext'
 import { SessionProvider } from '../../Context/SessionContext'
 import { MemoryRouter } from 'react-router-dom'
+import { UserProvider, UserContext } from '../../Context/UserContext'
 import { getSettings, updateSettings, getRandomContent, postSession } from '../../apiCalls'
 jest.mock('../../apiCalls.js')
 
 describe('App', () => {
-  it('should bring users to the timer view on load and allow them to visit the Stats, About, and Settings pages', async () => {
-
+  beforeEach(() => {
+    window.gapi = {
+      signin2: {
+        render: jest.fn(),
+      },
+    };
+  });
+
+  it('should render the login page on load', () => {
     getSettings.mockResolvedValue({
       data: {
         id: 1,
-        rest_interval: '5',
-        work_interval: '30',
-        sound: 'chordCliff'
-      }
+        rest_interval: "5",
+        work_interval: "30",
+        sound: "chordCliff",
```

Comment:
Can we update all instances of "chordCliff" to "reverbSplash"? I had meant to do that previously but I must have forgotten about the test files